app/src/main/java/com/seekerclaw/app/state/XaiOAuthDurabilityGate.kt

```
@@ -0,0 +1,168 @@
+        val live = !rec.tombstone && !rec.reauthRequired && rec.accessTokenEnc.isNotEmpty()
+        if (!live) return true
+
+        val deadlineNs = deadline()
+
+        // Phase 1 — give Node repeated chances to persist the VALID rotated T1.
+        var round = 0
+        while (round < MAX_FLUSH_ROUNDS && remainingMs(deadlineNs) > 0L) {
+            round++
+            val pending: Boolean? = try {
+                runBlocking { NodeControlClient.flushShutdown() }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)